src/ucs/stats/stats.h

```diff
@@ -14,16 +15,78 @@
 
 #include <ucs/sys/compiler_def.h>
 #include <ucs/sys/stubs.h>
+#include <ucs/stats/stats_fwd.h>
+
+#include "stats_fwd.h"
 
 BEGIN_C_DECLS
 
 /** @file stats.h */
 
+/* The maximum size of a class name */
+#define UCS_STATS_CLASS_NAME_MAX 50
+
+typedef struct {
+    /* Class base offset in the counters aggregate-sum list */
+    unsigned aggregate_sum_class_offset;
+
+} ucs_stats_aggrgt_sum_clsid_to_name_db_t;
```

Comment:
i think this struct is is not really needed, since it has only one field

Reply:
Removed.